### PR TITLE
[Functionality] - Add LinkedIn Share

### PR DESCRIPTION
**Pull request by MrSlimCoder**
Hi contributors , i have added linkedin share functionality in this project.
I'm very excited for your response.
Thanks !

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,56 +1,56 @@
...
+import { width, height } from "./utils/Dimensions";
+import { usePrevious } from "./utils/usePrevious";
+import { FaceBookIcon, TwitterIcon, LinkedinIcon } from "./assets/svgs";
+import HomeShowcase from "./components/HomeShocase";
+import SearchForm from "./components/SearchForm";
+import SocialHandler from "./components/SocialHandler";
+import ErrorMessage from "./components/ErrorMessage";
+import Profile from "./components/Profile";
+import GithubService from "./services/Github";
+import "./App.css";
```
Comment (NomanGul) [inline src/App.js:14]: remove semi-colon from changed files because we are using the no-semi-colon approach

File: src/assets/svgs.js [hunk 2]
```diff
@@ -28,4 +28,23 @@ export const TwitterIcon = () => (
...
+    <path
+      fill="#FFF"
+      d="M150.65,100.682c0,27.992-22.508,50.683-50.273,50.683c-27.765,0-50.273-22.691-50.273-50.683
+        C50.104,72.691,72.612,50,100.377,50C128.143,50,150.65,72.691,150.65,100.682z M143.294,187.333H58.277V462h85.017V187.333z
+        M279.195,187.333h-81.541V462h81.541c0,0,0-101.877,0-144.181c0-38.624,17.779-61.615,51.807-61.615
+        c31.268,0,46.289,22.071,46.289,61.615c0,39.545,0,144.181,0,144.181h84.605c0,0,0-100.344,0-173.915
+        s-41.689-109.131-99.934-109.131s-82.768,45.369-82.768,45.369V187.333z"
+    />
+  </svg>
+);
```
Comment (NomanGul) [inline src/assets/svgs.js:50]: is it possible to make the LinkedIn icon more round and responsive?

linkedIn issue

**Review by NomanGul — CHANGES_REQUESTED** [review]

**Comment by NomanGul** [comment]
@viveksharmaui please change your base branch from master to development

File: src/App.js (hunk 1 above)
Comment (MrSlimCoder) [reply src/App.js:14]: Sure i can remove them and Actually this happened because of prettier extension ..

File: src/assets/svgs.js (hunk 2 above)
Comment (MrSlimCoder) [reply src/assets/svgs.js:50]: Yeah i found this icon unresponsive i can update it .

**Comment by MrSlimCoder** [comment]
@NomanGul please add a contributing.md file so that new contributions know what to do and how to do things ..

**Comment by NomanGul** [comment]
> @NomanGul please add a contributing.md file so that new contributions know what to do and how to do things ..

@viveksharmaui done!

**Comment by MrSlimCoder** [comment]
See my update PR #35 
I'm closing this PR .

**Pull request closed without merging**